velox/exec/tests/utils/QueryAssertions.h

```diff
@@ -44,9 +44,8 @@ struct MaterializedRowComparator {
     return lhs.size() < rhs.size();
   }
 };
-// Multiset that uses 'epsilon' to compare doubles.
-using MaterializedRowMultiset =
-    std::multiset<MaterializedRow, MaterializedRowComparator>;
+// Multiset that compares floating-point values directly.
+using MaterializedRowMultiset = std::multiset<MaterializedRow>;
```

Comment:
As you found out the MaterializedRowComparator cannot be used as a comparator. Let's remove it to avoid confusion in the future.